setup.py

```diff
@@ -30,6 +32,7 @@
         'ckanapi.tests.mock',
         'ckanapi.cli',
         ],
+    setup_requires='requests',
```

Comment:
"Setuptools offers the setup_requires setup() keyword for specifying dependencies that need to be present in order for the setup.py script to run. Internally, Setuptools uses easy_install to fulfill these dependencies."